p2p/discv5/net.go

```diff
@@ -828,11 +827,10 @@ type nodeEvent uint
 //go:generate stringer -type=nodeEvent
 
 const (
-	invalidEvent nodeEvent = iota // zero is reserved
```

Comment:
I just noticed another issue here: the nodeEvent String method is created by stringer. Please re-run `go generate ./p2p/discv5` to update nodeevent_string.go.

Reply:
I'm running into a error when running stringer:
```go generate -v ./p2p/discv5
p2p/discv5/database.go
p2p/discv5/database_test.go
p2p/discv5/net.go
stringer: checking package: database.go:23:2: could not import bytes (reading export data: /usr/local/go/pkg/linux_amd64/bytes.a: invalid encoding format in export data: got 'v'; want 'c' or 'd')
p2p/discv5/net.go:827: running "stringer": exit status 1
```
I tried to find a way out, but so far I found nothing, any ideas ?

Reply:
@fjl sorry for the delay, I got caught in other issue to work on. Just re-generated the nodeevent with stringer as you requested.
Please let me know if anything else is missing.